### PR TITLE
[VIS] Fix scaling error in Transaction

### PR DESCRIPTION
**Pull request by Darrekt**
# Summary

Previously when games ran for too long, transactions would look like this

image

A simple scaling change fixes this in this PR. 

## Additional Information

`yarn.lock` was regenerated because the project wasn't building. Approvers may want to hold off on this PR until the out of memory issue gets fixed. 

This PR is proudly corona positive.

## Test Plan

- Add information on how you tested your changes.

### CODE REVIEW

**Comment by Eirikalb** [comment]
This PR is of great import! Love it <3

File: website/src/components/Visualisations/Transactions/Util/ProcessTransactionData.ts [hunk 1]
```diff
@@ -174,31 +174,33 @@ function processTransactionData(data: OutputJSONType) {
...
 
     const bubbleIds = [0, 1, 2, 3, 4, 5, 6]
+    let maxMag = 1000
+    const magnitudes = bubbleIds.map((team) =>
+        allTransactions
+            .filter(
+                (transaction) =>
+                    transaction.from === team || transaction.to === team
+            )
+            .reduce<number>(
```
Comment (lhl2617) [inline website/src/components/Visualisations/Transactions/Util/ProcessTransactionData.ts:184]: why is the type needed?

File: website/src/components/Visualisations/Transactions/Util/ProcessTransactionData.ts [hunk 2]
```diff
@@ -174,31 +174,33 @@ function processTransactionData(data: OutputJSONType) {
...
+                (acc, curr) =>
+                    curr.to === team ? acc + curr.amount : acc - curr.amount,
```
Comment (lhl2617) [inline website/src/components/Visualisations/Transactions/Util/ProcessTransactionData.ts:186]: can't you delete the filter and instead do 
```
reduce((acc, curr) => {
  if (curr.to === team) acc + curr.amount
  else if (curr.from === team) acc - curr.amount
}, 0)
```

File: website/src/components/Visualisations/Transactions/Util/ProcessTransactionData.ts [hunk 3]
```diff
@@ -174,31 +174,33 @@ function processTransactionData(data: OutputJSONType) {
...
-            return {
-                id: teamNo,
-                colorStatus: mag < 0 ? 'red' : 'green',
-                islandColor: thisTeamColor ?? '#031927',
-                magnitude: normaliseMag(Math.abs(mag), 125, 15, 1000, 0),
-            }
-        })
+    magnitudes.forEach((mag) => {
+        maxMag = mag > maxMag ? mag : maxMag
+    })
```
Comment (lhl2617) [inline website/src/components/Visualisations/Transactions/Util/ProcessTransactionData.ts:193]:
```
const maxMag = Math.max(1000, magnitudes.reduce(Math.max))
```
maybe `magnitudes.reduce((a, b) => Math.max(a, b))` required

**Review by lhl2617 — APPROVED** [review]
free to go--only nitpicks :)

**Pull request merged**